lib/plugins/fs_plugin.js

```diff
@@ -22,8 +22,9 @@ module.exports = class FsWebpackPlugin {
    * @param {Object} opts.matchers - matchers from spike config
    * @param {Array} opts.module.loaders - webpack loaders from spike config
    */
-  constructor (opts) {
+  constructor (opts, util) {
     this.opts = opts
+    this.util = util
```

Comment:
Ok so `util` actually holds on to a copy of the options, so if we are passing that in we might want to just pass in the util. Just have to make sure it is parallel with all the options needed.